contracts/optimistic-ethereum/iOVM/bridge/ERC20/iOVM_L2ERC20Gateway.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+import { iOVM_ERC20 } from "../../precompiles/iOVM_ERC20.sol";
+
+
+/**
+ * @title iOVM_L2ERC20Gateway
+ */
+interface iOVM_L2ERC20Gateway  /* TODO: add `is iOVM_ERC20` -- the types are messing with my tempERC20 */ {
```

Comment:
Nit: this todo makes the line too long, could easily go in the above comment below `@title`